xmodule/util/builtin_assets.py

```diff
@@ -12,6 +12,38 @@
 from openedx.core.djangoapps.theming.helpers_static import get_static_file_url
 
 
+def add_css_to_fragment(fragment, css_relative_path):
+    """
+    Given a css path relative to xmodule/assets, add a compiled CSS URL to the fragment.
+
+    Raises:
+    * ValueError if {css_relative_path} is absolute or does not end in '.css'.
+    * FileNotFoundError if edx-platform/xmodule/assets/{css_relative_path} is missing.
+    * ImproperlyConfigured if the lookup of the static CSS URL fails. This could happen
+      if CSS wasn't collected, or the staticfiles app is misconfigured.
+
+    Notes:
+    * This function is theme-aware. That is: If a theme is enabled which provides a compiled
+      CSS file of the same name, then that CSS file will be used instead.
```

Comment:
```suggestion
```
The function is not theme-aware. Regardless of theme, it should always return the same CSS file. If the theme makes any changes to video styling, it will go through the Sass variables in `_builtin-block-variables.scss`.